### PR TITLE
Add ConfigMap for latest EC task

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-1410

Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

File: components/enterprise-contract/kustomization.yaml [hunk 1]
```diff
@@ -11,6 +11,15 @@ resources:
...
 - https://raw.githubusercontent.com/hacbs-contract/enterprise-contract-controller/1985cbeddf7a803ab51597cae8cf4bfeca13acb4/config/rbac/enterprisecontractpolicy_editor_role.yaml
 
+generatorOptions:
+  disableNameSuffixHash: true
+
+configMapGenerator:
+- name: ec-defaults
+  namespace: enterprise-contract-service
+  literals:
+  - verify_ec_task_bundle=quay.io/hacbs-contract/ec-task-bundle:a98163a138491ba91ffce4e1f1853e3a52943f1e
```
Comment (lcarva) [inline components/enterprise-contract/kustomization.yaml:21]: This is the latest as of today. We'll use [HACBS-1411](https://issues.redhat.com/browse/HACBS-1411) to keep this up to date.

**Comment by lcarva** [comment]
/retest

I'm seeing this failure:

```
 • [FAILED] [140.232 seconds]
[release-suite [HACBS-1108]test-release-service-happy-path] Post-release verification. [It] The PipelineRun should exist and succeed. [release, HACBS]
/tmp/tmp.zYbluHpklG/tests/release/release.go:122
  [FAILED] Timed out after 120.001s.
  Expected
      <bool>: false
  to be true
  In [It] at: /tmp/tmp.zYbluHpklG/tests/release/release.go:131 @ 12/12/22 20:57:50.247 
```

Not sure if this is a random error...

**Review by zregvart — APPROVED** [review]

**Pull request merged**